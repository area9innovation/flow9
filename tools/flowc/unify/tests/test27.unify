import maybe;

α10004 = {None<α10002> .. *};


.(.value : α10003)  c=  α10004;

