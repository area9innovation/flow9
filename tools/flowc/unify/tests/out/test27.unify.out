α10004={None<α10002> .. *}
Expected .value in None
